### ISSUE TITLE
alias_method_chain deprecated and removed in Rails 5.1

### ISSUE THREAD

**Issue by oakesja** [adacosta/mongoid_rails_migrations#42]
When using this gem with Rails 5.0, the following deprecation warning appears

```
DEPRECATION WARNING: alias_method_chain is deprecated. Please, use Module#prepend instead. From module, you can access the original method using super.
```
It appears to be used [here](https://github.com/adacosta/mongoid_rails_migrations/blob/master/lib/mongoid_rails_migrations/active_record_ext/migrations.rb#L106)
`alias_method_chain` has also been removed in Rails 5.1. https://github.com/rails/rails/pull/27035

**Comment by francois2metz** [comment]
See #44

**Comment by jarthod** [comment]
Fixed in `1.1.1`
